src/main/java/bio/terra/cli/command/group/ListUsers.java

```diff
@@ -31,8 +36,49 @@ protected void execute() {
 
   /** Print this command's output in text format. */
   private static void printText(List<UFGroupMember> returnValue) {
-    for (UFGroupMember groupMember : returnValue) {
-      groupMember.print();
+    TablePrinter<UFGroupMember> printer = ListUsers.UFGroupMemberColumns::values;
```

Comment:
nit: you don't need to qualify `UFGroupMemberColumns` with `ListUsers` because it's the same class we're in now.